Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceListener.java

```diff
@@ -118,6 +119,12 @@ private void onIsolatePaused(@NotNull final IsolateRef isolateRef,
 
       final XLineBreakpoint<XBreakpointProperties> xBreakpoint = myBreakpointHandler.getXBreakpoint(vmBreakpoints.get(0));
 
+      if (xBreakpoint == null) {
+        myLatestSourcePosition = sourcePosition;
+        myDebugProcess.getSession().positionReached(suspendContext);
+        return;
+      }
```

Comment:
I guess this `if` is for stopping on breakpoints set in the Observatory, right?
If so, please make separate feature as a separate commit next time. Thanks!